File: 69-sqrtx/69-sqrtx.java
class Solution {
    public int mySqrt(int x) {
        if(x == 1){
            return 1;
        }
        int number = 0;
        for(int i = 0; i < x; ++i){
            if(((long) i * i) <= x){
                number = i;
            }
            else{
                break;
            }
        }

        return number;
    }
}

File: 69-sqrtx/69-sqrtx.c
int mySqrt(int x) {
	long num = 1;
	while (num <= x)
	{
		if (num * num == x) {
			return num;
		}
		else if (num * num > x) {
			return num - 1;
		}

		++num;
	}
	return 0;
}